File: app/src/main/java/com/example/heriprastio/recyclerviewinfragment/fragment/CurrentFragmet.java
package com.example.heriprastio.recyclerviewinfragment.fragment;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.example.heriprastio.recyclerviewinfragment.R;
import com.example.heriprastio.recyclerviewinfragment.adapter.MyAdapter;

public class CurrentFragmet extends Fragment {
    public CurrentFragmet() {
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView =  inflater.inflate(R.layout.fragment_current, container, false);
        RecyclerView mRecyclerView = (RecyclerView) rootView.findViewById(R.id.listRecycle);
        mRecyclerView.setHasFixedSize(true);
        MyAdapter adapter = new MyAdapter(new String[]{"Mummet 1", "Mummet 2", "Mummet 3", "Mummet 5", "Mummet 6", "Mummet 7"});
        mRecyclerView.setAdapter(adapter);
        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getActivity());
        mRecyclerView.setLayoutManager(linearLayoutManager);
        return rootView;

    }
}

File: app/src/main/java/com/example/heriprastio/recyclerviewinfragment/fragment/CurrentFragmet.kt
package com.example.heriprastio.recyclerviewinfragment.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.heriprastio.recyclerviewinfragment.R
import com.example.heriprastio.recyclerviewinfragment.adapter.MyAdapter
import com.example.heriprastio.recyclerviewinfragment.model.Team

class CurrentFragmet : Fragment() {

    private var teams: MutableList<Team> = mutableListOf()
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val rootView = inflater.inflate(R.layout.fragment_current, container, false)
        val mRecyclerView = rootView.findViewById<View>(R.id.listRecycle) as RecyclerView
        mRecyclerView.setHasFixedSize(true)
        val adapter = MyAdapter(teams)
        mRecyclerView.adapter = adapter
        val linearLayoutManager = LinearLayoutManager(activity)
        mRecyclerView.layoutManager = linearLayoutManager
        return rootView

    }
}
